cmd/cluster-kube-controller-manager-operator/render/render.go

```diff
@@ -20,27 +21,33 @@ const (
 type renderOpts struct {
 	manifest options.ManifestOptions
 	generic  options.GenericOptions
+
+	errOut io.Writer
 }
 
 // NewRenderCommand creates a render command.
-func NewRenderCommand() *cobra.Command {
-	renderOpts := &renderOpts{}
+func NewRenderCommand(errOut io.Writer) *cobra.Command {
+	renderOpts := &renderOpts{errOut: errOut}
 	cmd := &cobra.Command{
 		Use:   "render",
 		Short: "Render kubernetes controller manager bootstrap manifests, secrets and configMaps",
 		Run: func(cmd *cobra.Command, args []string) {
-			if err := renderOpts.Validate(); err != nil {
-				glog.Fatal(err)
-			}
-			if err := renderOpts.Run(); err != nil {
-				glog.Fatal(err)
+			must := func(fn func() error) {
```

Comment:
why not the error directly? Why the indirection with a func?

Reply:
@sttts error where? the Execute() won't propagate that error (Run does not return errors).

Reply:
```
must := func(error) {
  if err == nil {
    return
  }
  ....
}

must(renderOpts.Validate())
```